crates/uv/src/commands/tool/install.rs

```diff
@@ -297,10 +298,41 @@ pub(crate) async fn install(
         .collect::<BTreeSet<_>>();
 
     if target_entry_points.is_empty() {
+        match matching_packages(&from.name, &environment) {
+            Ok(packages) => {
+                if packages.is_empty() {
+                    writeln!(
+                        printer.stdout(),
+                        "The executable {} was not found.",
+                        from.name.red()
+                    )?;
+                } else {
+                    writeln!(
+                        printer.stdout(),
+                        "The executable {} was not found.",
+                        from.name.red()
+                    )?;
+
+                    let command = "uv tool install <PACKAGE>";
+                    writeln!(
+                        printer.stdout(),
+                        "However, the executable {} is available via {}:",
+                        from.name.green(),
+                        command.green(),
+                    )?;
+
+                    for package in packages {
+                        writeln!(printer.stdout(), "- {}", package.name().cyan())?;
+                    }
+                }
+            }
+            Err(err) => {
+                warn!("Failed to get entrypoints for packages in this environment: {err}");
```

Comment:
I think for user-facing copy we're using "executables"
```suggestion
                warn!("Failed to determine executables for packages in the environment: {err}");
```